runtime/src/main/java/org/corfudb/protocols/logprotocol/LogEntry.java

```diff
@@ -112,9 +111,7 @@ public boolean isMutation(UUID stream) {
     public enum LogEntryType {
         // Base Messages
         NOP(0, LogEntry.class),
-        SMR(1, SMREntry.class),
-        MULTIOBJSMR(7, MultiObjectSMREntry.class),
-        MULTISMR(8, MultiSMREntry.class),
+        SMRLOG(7, SMRLogEntry.class),
```

Comment:
Why not just SMR ?

Reply:
Because later we will have sth. like SMRGARBAGE for GC, in that case SMRLOG sounds more clear. And also a type different than the old SMR might be easier for later migration  tool.